time clock

string status
string addr
string port

RobotInfo[] active_robots

ZoneState[] zones

time tablet_last_beacon
bool tablet_display_map
time tablet_call_time
time tablet_position_time
float64 tablet_position_x
float64 tablet_position_y
